protocol/lib/metrics/constants.go

```diff
@@ -106,8 +106,11 @@ const (
 	ConvertToUpdates                             = "convert_to_updates"
 	CreateClobPair                               = "create_clob_pair"
 	Expired                                      = "expired"
+	FullyFilled                                  = "fully_filled"
 	GetFillQuoteQuantums                         = "get_fill_quote_quantums"
 	Hydrate                                      = "hydrate"
+	IsBuy                                        = "is_buy"
+	IsLong                                       = "is_long"
```

Comment:
Is `isBuy: true/false` going to be better than `side:bid/ask`?
Basically, consider a string tag rather than a boolean tag (which I think gets converted into a string anyway?)

Reply:
Got rid of `IsBuy` in favor of buy/sell.

Left `IsLong` as is since it's only used for deleveraging and is more appropriate.